ee/vellum_ee/workflows/display/nodes/vellum/prompt_deployment_node.py

```diff
@@ -43,9 +43,13 @@ def serialize(
         array_display = self.output_display[node.Outputs.results]
         json_display = self.output_display[node.Outputs.json]
 
-        deployment = display_context.client.deployments.retrieve(
-            id=str(raise_if_descriptor(node.deployment)),
-        )
+        if display_context.dry_run:
```

Comment:
instead of conditioning on `dry_run`, let's simply `try/except` the logic here and add any errors to the display context

Reply:
on the except case, return a uuid that is stable